tests/wpunit/Tribe/Validator/BaseTest.php

```diff
@@ -43,6 +43,28 @@ public function test_is_string( $value, $expected ) {
 		$this->assertEquals( $expected, $this->make_instance()->is_string( $value ) );
 	}
 
+	public function is_string_not_empty_data() {
+		return [
+			[ '', false ],
+			[ null, false ],
+			[ array( 'foo' => 'bar' ), false ],
+			[ array( 'foo', 'bar' ), false ],
+			[ new \StdClass(), false ],
+			[ 'f', true ],
+			[ 'foo bar', true ],
```

Comment:
Can we test for `0` as a number and a string?

Reply:
✅